backend/substrapp/views/computeplan.py

```diff
@@ -21,6 +24,9 @@ def create(self, request, *args, **kwargs):
         args = serializer.get_args(serializer.validated_data)
         try:
             ledger_response = query_ledger(fcn='createComputePlan', args=args)
+        except LedgerConflict as e:
+            error = {'message': str(e.msg), 'pkhash': e.pkhash}
+            return Response(error, status=e.status)
```

Comment:
I'm not sure what's the meaning of a conflict error for the compute plan, if it can happen and how we should handle it.
I don't think it makes sense to return to the API user a 'pkhash' in case of conflict.

Reply:
I can guarantee that it does happen :)

You can try to create the same compute plan multiple times in a row and you will get the conflict error. In that very specific case it makes sense to return the pkhash of the compute plan since you now can retrieve it.

That being said, I agree that the behavior is unclear when for example creating a sub-part of an already existing compute plan. But dealing with this is not part of this PR.

Reply:
Thanks for the clarification.